test/db-utils-test.js

```diff
@@ -301,253 +299,465 @@ describe("FaunaDB Utils", function () {
         });
     }).timeout(5000);
 
-    it("given a batch of scrapers outputs, can create, retrieve, and delete docs from Locations, ScraperRuns, and Appointments collections", async () => {
-        const timestamp = "2021-04-07T00:00:00.00Z";
-        const locations = [
-            {
-                name: `RandomName-${Math.random().toString(36).substring(7)}`,
-                street: "2240 Iyannough Road",
-                city: "West Barnstable",
-                zip: "02668",
-                availability: {
-                    "03/16/2021": {
-                        hasAvailability: true,
-                        numberAvailableAppointments: 2,
-                        signUpLink: "fake-signup-link-2",
+    it.skip(
```

Comment:
Skipping because we aren't using this batch logic, and I wanted to change some of the functions we in it. I plan to remove it in a future PR.